README.md

```diff
@@ -0,0 +1,38 @@
+<p align="center">
+  Alpha Merchant App - Simple Backend
+</p>
+
+
+## Description
+
+Simple backend to demonstrate how to implement the endpoints to integrate with Trustly.
+
+## Clone the app
+
+```bash
+# clone the repository
+$ git clone https://github.com/TrustlyInc/trustly-nestjs-example.git
+```
+
+## Configuring Trustly API credentials
+In the `.development.env` file replace environment variables values:
+- ACCESS_ID
+- ACCESS_KEY
+- BASE_URL
+
+## Install dependencies
+```bash
+# install dependencies
+$ cd trustly-nestjs-example && npm install
+```
+
+## Running the app
+```bash
+# development
+$ cd trustly-nestjs-example && npm run start:dev
```

Comment:
I'm still getting this error when I try to run the project from a fresh install and add my credentials to the env file:
<img width="868" alt="image" src="https://github.com/TrustlyInc/trustly-nestjs-example/assets/13320316/50b332f8-06da-4224-bff6-7a6c9ecbddf4">
Are there any prerequisites?

Reply:
Resolved. This project requires Node v16